CONTRIBUTING.md

```diff
@@ -8,30 +8,30 @@ To get started developing, see our [DEVELOPMENT.md](./DEVELOPMENT.md).
 
 In this file you'll find info on:
 
-* [Principles](#principles)
-* [The pull request process](#pull-request-process) and [Prow commands](#prow-commands)
-* [Standards](#standards) around [commit messages](#commit-messages) and [code](#coding-standards)
-* [Finding something to work on](#finding-something-to-work-on)
-* [The roadmap](#roadmap)
-* [Contacting other contributors](#contact)
+- [Principles](#principles)
+- [The pull request process](#pull-request-process) and [Prow commands](#prow-commands)
+- [Standards](#standards) around [commit messages](#commit-messages) and [code](#coding-standards)
+- [Finding something to work on](#finding-something-to-work-on)
+- [The roadmap](#roadmap)
+- [Contacting other contributors](#contact)
```

Comment:
why are `-` better than `*`? this seems random

Reply:
IDK, I was turned onto this tool by @evankanderson 

If I had to guess?  It is less ambiguous with bold / italics...?

Reply:
alright, `-` it is!